Transfer images (#1)
nebari-dev/nebari-docker-images: README.md

> # docker-images

Reviewer: This needs to be also a meaningful name 

```suggestion
# nebari base Docker images
```
 or something similar

> 🌅 Docker images for the Nebari platform

Reviewer: Needs replacing see below

> 

Reviewer: This readme went straight to the pinning... as an external contributor, I want to know the following:

1. The CI status - TODO: add badge for the CI
1. What is in this repo - i.e. the purpose of the images
2. How do I get started? what dependencies do I need? 
3. Step by step instructions to build the images because more than likely I will copy and paste in whichever order the instructions come in
4. Can I use them locally? if so how do I do this?
5. What is the license?
6. How do I report issues?
7. Where are the images pushed?

Inspiration:
- https://github.com/jupyterhub/repo2docker
- https://github.com/openshift/source-to-image
- https://github.com/jupyter/docker-stacks

Author: Added most of these items to the README 👍

# nebari base Docker images
🌅 Docker images for the Nebari platform

[![Build Docker Images](https://github.com/nebari-dev/nebari-docker-images/actions/workflows/build-push-docker.yaml/badge.svg)]

This repo contains the source code for Docker (container) images used by the Nebari platform. This repo also contains an automated means of building and pushing these images to public container registries. Currently these images are built and pushed to the following registries:

GitHub Container Registry (ghcr.io)
- [`nebari-jupyterlab`](https://github.com/orgs/nebari-dev/packages/container/package/nebari-jupyterlab)
- [`nebari-jupyterhub`](https://github.com/orgs/nebari-dev/packages/container/package/nebari-jupyterhub)

> - [nebari-dask-worker](https://github.com/orgs/nebari-dev/packages/container/package/nebari-dask-worker)

Reviewer: ```suggestion
- [`nebari-dask-worker`](https://github.com/orgs/nebari-dev/packages/container/package/nebari-dask-worker)
```

Quay Container Registry (quay.io)
- [`nebari-jupyterlab`](https://quay.io/repository/nebari/nebari-jupyterlab)
- [`nebari-jupyterhub`](https://quay.io/repository/nebari/nebari-jupyterhub)
- [`nebari-dask-worker`](https://quay.io/repository/nebari/nebari-dask-worker)

## Getting started

Whether you want to contribute to this project or simply wish use these images, to get started, fork this repo and then clone the forked repo onto your local machine.

> ### pre-requisite

Reviewer: ```suggestion
### Pre-requisite
```

keeping Title case

 
Currently the only pre-requisite is that you have [`docker` installed](https://docs.docker.com/get-docker/) on your local machine. 

### Building Dockerfile

> Assuming you are in the repos root folder, you can build these images locally by running the listed commands. 

Reviewer: ```suggestion
Assuming you are in the repo's root folder, you can build these images locally by running the listed commands on your terminal.
```

For JupyterLab 
```shell
docker build -f Dockerfile.jupyterlab -t qhub-jupyterlab:latest .
```

For JupyterHub
```shell
docker build -f Dockerfile.dask-worker -t qhub-dask-worker:latest .
```

For Dask-Worker
```shell
docker build -f Dockerfile.dask-gateway -t qhub-dask-gateway:latest .
```

> > NOTE: it is extremely important to pin specific packages `dask-gateway` and `distributed` as they need to be running the same version for the dask-workers to work as expected.

Reviewer: ```suggestion
> **NOTE** 
> It is extremely important to pin specific packages `dask-gateway` and `distributed` as they need to be running the same version for the dask-workers to work as expected.
```
